src/utilities/orientation/nearlyEqual.js

```diff
@@ -24,7 +24,7 @@ export default function nearlyEqual(a, b, epsilon) {
     } else if (a == 0 || b == 0 || absA + absB < Number.EPSILON) {
         // a or b is zero or both are extremely close to it
         // relative error is less meaningful here
-        return diff < epsilon * Number.EPSILON;
+        return diff < epsilon;
```

Comment:
If we have a constant (number.epsilon) and another epsilon variable, can we rename the variable to differentiate them? maybe this could help to avoid future confusion / new bugs.